docs/docs/index.md

```diff
@@ -5,10 +5,10 @@ sidebar_position: 1
 ---
 
 :::tip
-### **v17 Upgrade**
-Cosmos Hub will be upgraded to [v17.1.0](https://github.com/cosmos/gaia/releases/tag/v17.1.0) at block height: **[20,739,800](https://www.mintscan.io/cosmos/blocks/20739800)**
+### **Gaia v18 Upgrade**
+Cosmos Hub will be upgraded to [v18.0.0](https://github.com/cosmos/gaia/releases/tag/v18.0.0) at block height: [21,330,500](https://www.mintscan.io/cosmos/block/21330500).
 
-To upgrade from v16.0.0 check the [**upgrade guide**](https://github.com/cosmos/gaia/blob/release/v17.1.x/UPGRADING.md)
+To upgrade from v17.3.0 check the [upgrade guide](https://github.com/cosmos/gaia/blob/v18.0.0/UPGRADING.md).
```

Comment:
```suggestion
To upgrade from v17.2.0 check the [upgrade guide](https://github.com/cosmos/gaia/blob/release/v18.x/UPGRADING.md).
```
The upgrade instruction from v17.3.0 will be part of https://github.com/cosmos/gaia/blob/release/v18.1.x/UPGRADING.md and we will add them to the banner once v18.1.0 is out. cc @sainoe